### PR TITLE
✨Add emojis to review template and clarify ✨

### PR DESCRIPTION
**Pull request by nikolaik**
This adds some emojis, adding distinct colors to section headers, and moves the checklist to the bottom of the list, since it most often is irrelevant to the reviewer

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
 <!--- Provide a general summary of your changes in the title. -->
 
-## Summary
-<!--- Use this section to describe your changes. Link this PR to relevant Trello cards. If your PR is still a work in progress, create PR as draft. -->
+## 📝 What's changed
```
Comment (nikolaik) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
## 📝 Summary
```

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
+## ✨ Screenshots/GIF's
+<!-- If the PR introduces a new feature or enhancement of existing feature, you can add screenshots or GIF's showcasing the changes (with e.g. https://getkap.co/). -->
 
-- [ ] I have assigned a reviewer.
-- [ ] I have added relevant labels.
-- [ ] (_after creating the PR_) I have added a new card to the Trello board and attached this PR.
-- [ ] I have checked that this PR supports [WCAG 2.0](https://uu.difi.no/krav-og-regelverk/wcag-20-standarden)
+## 💁‍♂️ Reviewer instructions
+<!-- Don't forget to assign a reviewer to the PR! Use this section if:
+- There is something the reviewer needs to know before make their review
```
Comment (nikolaik) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
- There is something the reviewer needs to know before reviewing
```

**Comment by henrikq** [comment]
Can we have a reminder in the checklist to add a emoji as well?
ref.
https://github.com/convertelligence/kindly/pull/2187#issuecomment-555389290

**Comment by nikolaik** [comment]
> Can we have a reminder in the checklist to add a emoji as well? pretty plz :)
> ref.
> [convertelligence/kindly#2187 (comment)](https://github.com/convertelligence/kindly/pull/2187#issuecomment-555389290)

Sure thing, I'll add it to the checklist

**Pull request merged**